### PR TITLE
[STK-114] Add bootstrap.sh and update dockerfile

### PR DESCRIPTION
**Pull request by waseem-h**
This PR updates the dockerfile and adds a bootstrap.sh file which is executed upon container creation

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,11 +4,10 @@ LABEL author="stakater"
 ARG SRC_DIR="ingress-monitor-controller"
 ENV SRC_DIR=${SRC_DIR}
 
+ADD bootstrap.sh /bootstrap.sh
+
+RUN chmod +x /bootstrap.sh
```
Comment (hazim1093) [inline Dockerfile]: No need to run this here, run this locally on the script so the file is committed with executable permissions

File: bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
```
Comment (hazim1093) [inline bootstrap.sh]: Add "echos" just to tell what's happening so the logs are understandable

File: Dockerfile (hunk 1 above)
Comment (waseem-h) [reply Dockerfile]: done

File: bootstrap.sh (hunk 2 above)
Comment (waseem-h) [reply bootstrap.sh]: done

**Pull request merged**